views/partials/what_to_do_svg.hbs

```diff
@@ -11,7 +10,7 @@
       <g transform="translate(118.000000, 98.000000)">
         <circle stroke="url(#linearGradient-1)" stroke-width="2" cx="20" cy="20" r="20"></circle>
```

Comment:
Not sure if this should be `url(#linearGradient-{{ number }})` to match L3 above (unless all 4-ish gradients are the same and it doesnt matter...).

Reply:
They're all the same but worth changing lest #linearGradient-1 be lurking around unnoticed elsewhere and for the sake of being thorough.